### ISSUE TITLE
docs: redirect root pages to the latest version

### ISSUE THREAD

**Issue by XhmikosR** [twbs/bootstrap#27904]
Like `/about/` to `4.2/about/` etc.

Here's the list

```
about
browser-bugs
components
css
customize
examples
    blog
    carousel
    cover
    dashboard
    grid
    jumbotron
    jumbotron-narrow
    justified-nav
    navbar
    navbar-fixed-top
    navbar-static-top
    non-responsive
    offcanvas
    signin
    starter-template
    sticky-footer
    sticky-footer-navbar
    theme
    tooltip-viewport
getting-started
javascript
migration -> to /docs/3.4/migration/ ?
```

**Comment by XhmikosR** [comment]
Here's the list

```
about
browser-bugs
components
css
customize
examples
    blog
    carousel
    cover
    dashboard
    grid
    index.html
    jumbotron
    jumbotron-narrow
    justified-nav
    navbar
    navbar-fixed-top
    navbar-static-top
    non-responsive
    offcanvas
    signin
    starter-template
    sticky-footer
    sticky-footer-navbar
    theme
    tooltip-viewport
getting-started
javascript
migration -> to /docs/3.4/migration/
```

**Comment by mdo** [comment]
Took a first pass at thinking through what each should do. Let me know what you think.

| Page | What to do |
| --- | --- |
| `about` | Redirect to the latest version in `docs/x.x/` release |
| `browser-bugs` | Redirect to the latest version in `docs/x.x/` release |
| `components` | Redirect to the latest version in `docs/x.x/` release |
| `css` | No redirect |
| `customize` | No redirect |
| `examples` | Redirect to the latest version in `docs/x.x/` release |
| Individual examples | _Don't redirect? Unsure..._ |
| `getting-started` | Redirect to the latest version in `docs/x.x/` release |
| `javascript` | No redirect |
| `migration` | Redirect to the latest version in `docs/x.x/` release |

**Comment by XhmikosR** [comment]
I'm gonna make a patch for these, except the no redirect ones. I want to tackle those separately because I know we reference them somewhere, at least `css` and `javascript`.

**Comment by XhmikosR** [comment]
So, we still need to redirect the individual examples. I'm gonna fix it later and redirect them to the 3.4 examples.

**Comment by XhmikosR** [comment]
#27961 was merged.
